ultrack/core/export/_test/test_exporter.py

```diff
@@ -1,10 +1,12 @@
 from pathlib import Path
 
+import pytest
+
 from ultrack import MainConfig, export_tracks_by_extension
 
 
 def test_exporter(tracked_database_mock_data: MainConfig, tmp_path: Path) -> None:
-    file_ext_list = [".xml", ".csv", ".zarr", ".dot", ".json"]
+    file_ext_list = [".xml", ".csv", ".zarr", ".dot", ".json", ".geff.zarr"]
```

Comment:
```suggestion
    file_ext_list = [".xml", ".csv", ".zarr", ".dot", ".json", ".geff"]
```
What do you think of this?